gdal/ogr/ogrsf_frmts/cad/GNUmakefile

```diff
@@ -0,0 +1,20 @@
+
+include ../../../GDALmake.opt
+
+SUBDIRS-yes := libopencad
+
+OBJ	= \
+	ogrcaddriver.o \
+	gdalcaddataset.o \
+	ogrcadlayer.o \
+	vsilfileio.o
+
+CPPFLAGS	:=	-I. -I.. -I../.. $(CPPFLAGS) -std=c++11 
```

Comment:
C++11 is absolutely necessary. I have added a test in configure of compilator compability with -std=c++11, but cant understand what to do to get variable CAD_ENABLED in ogrdrivers makefile.